### PR TITLE
fix: make sure paths are consistent from os to os

### LINKED ISSUE #60: Option to map entries or normalize paths on both Windows and OSX/Linux
Hi,

The keys of the manifest on Windows are: `"path\\file.ext"`

Whereas on Linux/OSX it is: `"path/file.ext"`

Making it difficult to reference a resource by its key in the manifest.

Can we have a callback function to transform the manifest elements prior to insertion or have both paths use slashes (`/`)?

### PR DESCRIPTION
**Pull request by mastilver**
closes #60

### CODE REVIEW

File: lib/plugin.js [hunk 1]
```diff
@@ -110,6 +110,12 @@ ManifestPlugin.prototype.apply = function(compiler) {
       }.bind(this));
     }
 
+    files = files.map(file => {
+      file.name = file.name.replace('\\', '/');
+      file.path = file.path.replace('\\', '/');
```
Comment (a-x-) [inline lib/plugin.js]: `'\\'` is equivalent `/\//`
but `/\//g` expected

Comment (a-x-) [reply lib/plugin.js]:
```js
'sdfds\\sdfsdf\\'.replace('\\', '/') // -> "sdfds/sdfsdf\"
'sdfds\\sdfsdf\\'.replace(/\\/, '/') // -> "sdfds/sdfsdf\"
'sdfds\\sdfsdf\\'.replace(/\\/g, '/') // -> "sdfds/sdfsdf/"
```

File: spec/plugin.spec.js [hunk 2]
```diff
@@ -398,6 +398,22 @@ describe('ManifestPlugin', function() {
         done();
       });
     });
+
+    it('should output unix paths', function(done) {
+      webpackCompile({
+        context: __dirname,
+        entry: {
+          'dir\\main': './fixtures/file.js'
```
Comment (a-x-) [inline spec/plugin.spec.js]: add `some\\dir\\main also`

File: lib/plugin.js (hunk 1 above)
Comment (mastilver) [reply lib/plugin.js]: Good catch :+1:

**Comment by mastilver** [comment]
@a-x- fixed the issue you spotted :)

**Pull request merged**